pkg/controller/apiextensions/composite/reconciler.go

```diff
@@ -56,8 +56,8 @@ const (
 	errConfigure    = "cannot configure composite resource"
 	errPublish      = "cannot publish connection details"
 	errRender       = "cannot render composed resource"
-
-	errFmtRender = "cannot render composed resource at index %d"
+	errFmtRender    = "cannot render composed resource at index %d"
+	errInjectCD     = "cannot inject Composition data onto composite resource"
```

Comment:
Nit: I think the intent here was to keep `errFmt` and `err` strings separated, so I'd put this up under `errRender`.